src/dataverk/utils/anonymization.py

```diff
@@ -51,8 +93,11 @@ def _check_additional_columns_type(additional_columns):
 
 def _check_column_names(df, columns):
     for column in columns:
+        if not isinstance(column, str):
+            raise TypeError(f"{column}: column names should be of type str")
```

Comment:
Spørsmål: Ønsker vi et absolutt krav om at column name må være string. Kan kanskje tenkes scenarior hvor kolonnenavn ikke er string i dataframen.

Reply:
Tenkte ikke over at kolonnenavn kunne være av flere typer. Trenger ikke begrense, men blir noen typer som vil gi feilmelding når en prøver å sjekke om de finnes i kolonnelista (liste, sett, dictionary). Om disse er nødvendige å sjekke for blir et nytt spørsmål:)